lib/utilities/paginator.rb

```diff
@@ -0,0 +1,85 @@
+module Utilities
+  module Paginator
+    def items_to_display(item:, items_per_page:)
+      @items_per_page = items_per_page || 5
+      @current_page = params[:page] ? params[:page].to_i : 1
```

Comment:
I dont like this class looking at the params. I think the current page should be passed in